server/matchmaker/matchmaker_queue.py

```diff
@@ -48,7 +49,9 @@ def __init__(
         self._is_running = True
 
         self.timer = PopTimer(self.queue_name)
-        asyncio.ensure_future(self.queue_pop_timer())
+        if not loop:
+            loop = asyncio.get_running_loop()
+        loop.create_task(self.queue_pop_timer())
```

Comment:
same here, why not to `await`?

Reply:
Imho, it's our opportunity to migrate everything to await, reason for `create_task` for me is only "run from nonasync code", otherwise it's almost the same, but more readable.

Reply:
Because this is in `__init__` it is not async

Reply:
Also in this case `create_task` is what we want. We can't `await` it because the coroutine runs until the server is shut down.